src/core/fs.h

```diff
@@ -84,11 +90,23 @@ fs_basedir(
     size_t maxlen)  // allocation size
 {
 #ifdef __linux__
+  #ifdef LIBVKDT
+  void *handle;
+  char mod[PATH_MAX];
+  handle = dlopen("libvkdt.so", RTLD_LAZY);
```

Comment:
are you leaking this? how does dlopen work? is it reference counted so that you can safely call dlclose and the main dynamic linkage will not disappear?

Reply:
Yeah, in a hurry to post before bedtime, forgot to include a dlclose() call.